### DISCUSSION TITLE
White theme

### DISCUSSION THREAD

**Discussion by AlexanderRU44** [maxrave-dev/SimpMusic#103] [Ideas]
Will there be a light light theme?

**Comment by maxrave-dev** [comment]
I don't know but personally, I love dark mode. But if this discussion have many upvote, I will try to add light theme. Cheers 🥂

**Reply by ankitshekhawat21** [reply]
Everybody loves dark mode , but it is not visible in sunny outdoors on IPS screen phones( which is what atleast 60 percent users have).
